crates/wallet/src/lib.rs

```diff
@@ -251,82 +243,59 @@ where
         // we acquire the permit here so that all following operations are performed exclusively
         let _permit = self.inner.permit.lock().await;
 
-        // set nonce
-        let next_nonce = LoadState::next_available_nonce(
-            &self.inner.eth_api,
-            NetworkWallet::<Ethereum>::default_signer_address(&self.inner.wallet),
-        )
-        .await
-        .map_err(|err| {
-            self.inner.metrics.invalid_send_transaction_calls.increment(1);
-            err.into()
-        })?;
-        request.nonce = Some(next_nonce);
-
         // set chain id
         request.chain_id = Some(self.chain_id());
 
         // set gas limit
         // note: we also set the `from` field here to correctly estimate for contracts that use e.g.
         // `tx.origin`
-        request.from = Some(NetworkWallet::<Ethereum>::default_signer_address(&self.inner.wallet));
+
+        // todo: what to do here
+        // request.from =
+        // Some(NetworkWallet::<Ethereum>::default_signer_address(&self.inner.wallet));
```

Comment:
How do I get the address of the signer in an alloy provider? cc @mattsse perhaps or @klkvr

Reply:
resolved